### DISCUSSION TITLE
Resource Menu not showing in Admin Panel.

### DISCUSSION THREAD

**Discussion by bahadur** [filamentphp/filament#12938] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.0

### How can we help you?

I have created two panels, 'admin' and 'customer'. Under the admin panel, I created a resource for Customer. However, when I log in to the admin panel, the Customer menu doesn't appear. Please help me with this.

**Comment by bahadur** [comment]
I have noticed that when I change the ID to something other than 'admin', the resources appear in the menu. However, if the ID is 'admin', the resources do not show up in the menu. 
What could be the reason?

**Comment by shebaoting** [comment, marked as answer]
```
php artisan filament:clear-cached-components
```

**Reply by mack-cs** [reply]
This worked for me

**Reply by johnymanuelli** [reply]
Thanks!!!

**Reply by enisabazi2005** [reply]
Thanks man, it worked for me

**Reply by putricha** [reply]
Thanks!! This worked for me

**Reply by alvinChristianto** [reply]
Thanks! I though something wrong with filament-shield conf.

**Reply by Mzati1** [reply]
Thanks!

**Reply by fernaerell** [reply]
Thank u bro!

**Reply by ganjargal** [reply]
Thanks!
